NetworkService/Sources/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,8 @@
+final class NetworkService {
+    /// Stub method to check possibility to use this NetworkService in your App.
+    ///
+    ///  - Returns: true
+    func use() -> Bool {
```

Comment:
Could you please tell me what access level has the current method?

Reply:
Internal access.

Reply:
What access level we need for package's API?

Reply:
public